README.md

```diff
@@ -1,57 +1,72 @@
+## Usage Examples
+Use the **Run MATLAB Tests** action to automatically run tests authored using the MATLAB Unit Testing Framework or Simulink Test&trade;. You can use this action with optional inputs to generate different types of test artifacts.
 
-## Example
+### Run MATLAB Tests on Self-Hosted Runner
+Use a self-hosted runner to automatically run the the tests in your [MATLAB project](https://www.mathworks.com/help/matlab/projects.html).
 
-```yaml
-name: Sample workflow
+```yaml 
+name: Run MATLAB Tests on Self-Hosted Runner
 on: [push]
+jobs:
+  my-job:
+    name: Run MATLAB Tests
+    runs-on: self-hosted
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+      - name: Run tests
+        uses: matlab-actions/run-tests@v0
+```
+
+### Generate Test Artifacts on GitHub-Hosted Runner
+Use the [Set Up MATLAB](https://github.com/matlab-actions/setup-matlab/) action when you want to run tests and generate artifacts on a GitHub-hosted runner. The action installs your specified MATLAB release (R2020a or later) on a Linux virtual machine. If you do not specify a release, the action installs the latest release of MATLAB.
 
+For example, install the latest release of MATLAB on a GitHub-hosted runner, and then use the **Run MATLAB Tests** action to run the tests in your MATLAB project and generate a JUnit test results report and a Cobertura code coverage report.
+
+```yaml
+name: Generate Test Artifacts on GitHub-Hosted Runner
+on: [push]
 jobs:
   my-job:
-    name: Run MATLAB Tests and Save Results
+    name: Run MATLAB Tests and Generate Artifacts
     runs-on: ubuntu-latest
     steps:
-      # Checkout the project from GitHub
-      - uses: actions/checkout@v2
-
-      # Set up MATLAB using this action first if running on a GitHub-hosted runner!
-      - uses: matlab-actions/setup-matlab@v0
-      
-      # Run the MATLAB tests inside the repo and produce test artifacts
-      - name: Run all the tests
+      - name: Check out repository
+        uses: actions/checkout@v2
+      - name: Install MATLAB
+        uses: matlab-actions/setup-matlab@v0
+      - name: Run tests and generate artifacts
         uses: matlab-actions/run-tests@v0
         with:
-            test-results-junit: test-results/results.xml
-            code-coverage-cobertura: code-coverage/coverage.xml
+          test-results-junit: test-results/results.xml
+          code-coverage-cobertura: code-coverage/coverage.xml
 ```
+## Run MATLAB Tests
+When you define your workflow in the `.github/workflows` directory of your repositoy, you can specify the **Run MATLAB Tests** action using the `run-tests` key. The action accepts optional inputs.
```

Comment:
Same as `run-command`: it's unlikely that a user would use `run-tests` unqualified without our org/version 🤔

Reply:
Thanks, Johan. Does the updated text help?